languages/swift/maintainers.md

```diff
@@ -6,6 +6,8 @@ These awesome people help maintain the Swift track.
 
 TODO: add senior maintainers -->
 
+- Haunshila Yadav (Slack: `@haunshila`, Github: [haunshila](https://github.com/haunshila))
```

Comment:
Could you list yourselves in the `## Contributing Maintainers` section please?

Reply:
@ErikSchierboom Done